x-pack/test/security_solution_cypress/cypress/e2e/investigations/threat_intelligence/indicators.cy.ts

```diff
@@ -270,6 +269,7 @@ describe('Multiple indicators', { tags: ['@ess'] }, () => {
     });
 
     it('should handle all search actions', () => {
+      // TODO fix
```

Comment:
if this is still a todo, could you add an issue reference here?

Reply:
crap, I forgot to remove it, that was just for me to remember to check the tests 😆

Reply:
[removed](https://github.com/elastic/kibana/pull/195826/commits/7980e650715fd4128ff5b89e612bd7b08828940a)